addons/core/translations/resources/en-us.yaml

```diff
@@ -418,6 +409,7 @@ credential:
   types:
     username_password: Username & Password
     ssh_private_key: Username & Key Pair
+    unknown: Unknown
```

Comment:
Should this maybe be in a more generic place since it looks like both `credential-library` and `credential` uses this and I assume more in the future?

Reply:
Yes I agree. I don't think it is blocking for this PR since we have a large translation refactor coming up soon. We need a more generic term for referring to both credential libraries and static credentials.

Reply:
yes, we need to come up with a generic term and a spot where we can add all cred/cred library-related types.